custom_components/rpi_gpio/hub.py

```diff
@@ -199,7 +199,8 @@ def add_sensor(self, entity, port, active_low, bias, debounce) -> None:
                 bias = BIAS[bias],
                 active_low = active_low)},
         ) as request:
-            entity._attr_is_on = True if request.get_value(port) == Value.ACTIVE else False
+            # Although we prefer to set the _attr_is_on attribute, in cover it does not exists for whatever reason.
+            entity.is_on = True if request.get_value(port) == Value.ACTIVE else False
```

Comment:
@tomer-w Please do not merge PRs without review, this comment is irrelevant, you are setting a variable inside the `hub` class and any `_attr` is not relevant here, I missed this on the previous review.

Reply:
Not an expert, but for ```binary_sensor``` this code with the ```_attr``` is working fine. This was tested by me before the original submit. This is something which is broken only for the ```cover``` case.
With regards to the pushing it without waiting for your PR, I didnt want to keep customers with broken version, but of course I will follow on whatever CR comment you will have.